File: catkin_ws/src/my_robot_tutorials/src/my_first_node.cpp
#include <ros/ros.h>

int main(int argc, char **argv)
{
    ros::init(argc, argv, "my_first_cpp_node");
    ros::NodeHandle nh;

    ROS_INFO("Node has been started");

    ros::Rate rate(1);

    int count = 1;
    while(ros::ok())
    {
        ROS_INFO("Hello %i", count);
        count += 1;
        rate.sleep();
    }
}

File: catkin_ws/src/my_robot_tutorials/scripts/my_first_node.py
#!/usr/bin/env python


import rospy

if __name__ == "__main__":
    rospy.init_node("my_first_python_node")

    rospy.loginfo("This node has been started")
    count = 1
    script_rate = rospy.Rate(1)
    while not rospy.is_shutdown():
        rospy.loginfo(f"Hello {count}")
        count+=1
        script_rate.sleep()
    